src/types/GitHubRateLimitResource.Types.ps1xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Types>
+    <Type>
+        <n>GitHubRateLimitResource</n>
+        <Members>
+            <ScriptProperty>
+                <n>ResetAt</n>
+                <GetScriptBlock>$this.Reset</GetScriptBlock>
+            </ScriptProperty>
+            <ScriptProperty>
+                <n>ResetIn</n>
+                <GetScriptBlock>
+                    if ($null -eq $this.Reset) { return }
+                    $timeRemaining = $this.Reset - [DateTime]::Now
+                    if ($timeRemaining.TotalSeconds -lt 0) {
+                        return [TimeSpan]::Zero
+                    }
+                    return $timeRemaining
+                </GetScriptBlock>
```

Comment:
Make the TimeSpan a class. But we want to format it to pretty print timespan here.